modules/otel-troubleshoot-instrumentation.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * observability/otel/otel-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="troubleshooting-instrumentation_{context}"]
+= Troubleshooting instrumentation
+
+You can troubleshoot instrumentation issues as follows.
+
+.Procedure
+
+. Check the instrumentation object is created:
++
+[source,console]
+----
+oc get instrumentation -n <your-project>
+----
+
+. Confirm that the application pods you want to instrument are labeled correctly and the appropriate auto-instrumentation settings have been applied. For example, if you are instrumenting a Python application, you can check that the pods have the `instrumentation.opentelemetry.io/inject-python='true'` label:
++
+
+[source,console]
+----
+oc get pods -n <your-project> -o jsonpath='{range .items[?(@.metadata.annotations["instrumentation.opentelemetry.io/inject-python"]=="true")]}{.metadata.name}{"\n"}{end}'
+----
+
+You should double-check the annotation applied to the instrumentation object is the correct one for the language you are instrumenting.
```

Comment:
We can probably use more troubleshooting points from https://opentelemetry.io/docs/kubernetes/operator/automatic/ Also if the CR is in different namespace, we need to specify the namespace in the instrumentation annotation. If there are multiple CRs, we need to specify the CR name in the annotation. https://opentelemetry.io/docs/kubernetes/operator/automatic/#add-annotations-to-existing-deployments 

We can also add troubleshooting point to use the debug exporter to check if the traces are being sent from the instrumented app.